test/integration/Service.spec.ts

```diff
@@ -105,6 +105,31 @@ describe('API Service', () => {
     expect(order.id).to.equal(orderId);
   });
 
+  it('should list order history with a trade', async () => {
+    const { status, trades, swapDeals } = await service.listOrderHistory({ orderId: orderId! });
+
+    expect(status).to.equal(2);
+    expect(trades).to.eql([]);
+    expect(swapDeals).to.eql([]);
```

Comment:
Just for legibility purposes, I think it'd be nicer if these all ready `to.equal` instead of `to.eql` so there's no doubt that they're doing the same thing.

Reply:
in fact they're doing different things, `eql` is checking deeply as far as I remember, if I use equal it throws with `not equal` error